gui/mozregui/wizard.py

```diff
@@ -111,6 +111,27 @@ def __init__(self):
         self.ui.start_date.setDateTime(now.addYears(-1))
         self.ui.end_date.setDateTime(now)
 
+    def validatePage(self):
+        good_date = self.ui.start_date.date()
+        bad_date = self.ui.end_date.date()
+        current = QDateTime.currentDateTime().date()
+        if good_date < bad_date:
+            if bad_date <= current:
+                return True
+            else:
+                QMessageBox.critical(
+                    self,
+                    "Error",
+                    "Date you entered is larger than now,please try it again.")
```

Comment:
I'm not sure the message here is great english (I'm not a natural english speaker neither but larger for a date seems strange to me). Also I find the "please try again" not really useful.
Maybe "You can't define a date in the future." would be enough ?